compose/reference/scale.md

```diff
@@ -13,3 +13,10 @@ Sets the number of containers to run for a service.
 Numbers are specified as arguments in the form `service=num`. For example:
 
     docker-compose scale web=2 worker=3
+
+>**Tip:** Alternatively, in
+[Compose file version 3.x](/compose/compose-file/index.md)
+with Docker Engine 1.13 and above, you can specify
```

Comment:
"Engine 1.13" is redundant since we're already talking about "Compose file version 3.x"